### PR TITLE
FORD-115: highlight and display currently selected video on player load

### PR DESCRIPTION
**Pull request by BC-Mikey**

### CODE REVIEW

File: lib/bcgs_ford_filmstrip.js [hunk 1]
```diff
@@ -87,6 +87,13 @@ v1.0.0: 6/19/2015
                 if (!bcc.model.getConfig().defaultFilmstripExpanded) {
                     onFilmstripToggle();
                 }
+                
+                // highlight and scroll to initially selected video
+                carousel.index = bcc.model.getConfig().playIndex;
```
Comment (tyard) [inline lib/bcgs_ford_filmstrip.js]: I think you need to look at the playlist.getSelectedIndex() as opposed to the main model. That index will be set to whatever it should be (that's how the player is getting its video). Doing it this way means a) the two could potentially be out of sync and b) a default value will be ignored.

Comment (BC-Mikey) [inline lib/bcgs_ford_filmstrip.js]: Fixed in new commit.

**Comment by tyard** [comment]
looks good

**Comment by BC-Mikey** [comment]
All merges made and ready to merge to dev

**Pull request merged**